schema.sql

```diff
@@ -0,0 +1,10 @@
+/* create the animal table for database */
+CREATE TABLE animals (
+  id BIGSERIAL PRIMARY KEY,
+  name VARCHAR(50),
+  date_of_birth DATE,
+  escape_attempt INT,
+  neutered BOOLEAN,
+  weight_kg DECIMAL,
+)
+
```

Comment:
### Optional
- [ ] It would be better if we make make the id column not null and autoincremented